packages/client-config/src/client-config-writer/client_config_to_mobile_legacy_converter.test.ts

```diff
@@ -181,6 +181,7 @@ void describe('client config converter', () => {
                 mfaConfiguration: undefined,
                 mfaTypes: undefined,
                 signupAttributes: [],
+                socialProviders: [],
```

Comment:
We should have a test where this is not just an empty array

Reply:
Done 27cfb48

Reply:
And done [05db727](https://github.com/aws-amplify/amplify-backend/pull/1655/commits/05db727bbaf4a7161fe1e613a9e96e8c488a849f)